Boards: drag and drop (#6106)
mattermost/docs: source/boards/navigate-boards.rst

.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
  :start-after: :nosearch:
  
.. |options-icon| image:: ../images/dots-horizontal_F01D8.svg

>   :height: 24px

Reviewer: You can safely remove the height/width values for compass design elements. The default size of these images works well for the docs.

.. note::

  Upgrading to Mattermost Boards v7.2 onwards: If you belonged to a workspace prior to v7.2, you’ll see that the workspaces have been migrated to custom categories in the sidebar. All boards from a workspace are listed under a category of the same name. Boards from direct messages and group messages appear under the default **Boards** category.
  
  Categories are per-user, and can be renamed or deleted by each user after migration. New users won’t have default categories, and boards they join will appear under the default **Boards** category.

  Boards that you create after the migration won’t be linked to a workspace and will always appear under the default **Boards** category unless you move or hide the boards.
  
Drag and drop
-------------

> You can move both sidebar categories and boards and change their order to suit your preference. You can:

Reviewer: ```suggestion
You can move both sidebar categories and boards and change the order of both to suit your preference. You can:
```

- Set the position of a board within a category.
- Drag a board out of one category and drop it into another category.

> To do this, select and hold the cursor over the category or board name. Then move the category or board around as needed. Boards moved into a category are sorted on top of the category by default unless you specifically position the board before releasing the cursor.

Reviewer: ```suggestion
To do this, select and hold the cursor over the category or board name. Then move the category or board around as needed. Boards moved into a category are sorted to the top of the category by default unless you specifically position the board before releasing the cursor.
```

Reviewer: This may be an excellent candidate for an animated GIF approach (now or in a future iteration as time permits).

> Bulk drag and drop boards

Reviewer 2: @justinegeffen I don't think bulk drag and drop is supported in the release. I am unable to do any bulk actions on Chrome. @harshilsharma63 Can you confirm? Thanks!

~~~~~~~~~~~~~~~~~~~~~~~~~

> You can select multiple boards on the sidebar and drag them into a new category or position. Bulk select only applies to boards and not categories

Reviewer: ```suggestion
You can select multiple boards on the sidebar and drag them into a new category or position. Bulk select only applies to boards and not categories.
```

There are two options available:

> - **Sequentially:** Select and hold the cursor, then press the SHIFT key. This selects boards sequentially below the first select board.

Reviewer: Please use the ``:kbd:`` format for these keyboard shortcuts, and include both the Windows/Linux and Mac equivalents (now or in a later revision as time permits)
